dask/dataframe/core.py

```diff
@@ -2992,6 +2998,9 @@ def isin(self, values):
     @insert_meta_param_description(pad=12)
     @derived_from(pd.Series)
     def map(self, arg, na_action=None, meta=no_default):
+        """
+        Note that the index and divisions are assumed to remain unchanged.
```

Comment:
This warning just needs to apply to Index.map (which subclasses Series). I don't think that Series.map can change the index values.

Reply:
ah ok. I misunderstood.